Xamarin.Essentials/Types/DisplayInfo.shared.cs

```diff
@@ -5,13 +5,14 @@ namespace Xamarin.Essentials
     [Preserve(AllMembers = true)]
     public readonly struct DisplayInfo : IEquatable<DisplayInfo>
     {
-        public DisplayInfo(double width, double height, double density, DisplayOrientation orientation, DisplayRotation rotation)
+        public DisplayInfo(double width, double height, double density, DisplayOrientation orientation, DisplayRotation rotation, float rate)
```

Comment:
We can't break the API, so we need to add an overload with this new parameter.

Reply:
Good point. I did not think about this. In fact I considered the `DisplayInfo` constructor to be more of an 'internal' API, since it is usually not called directly by the end user. In a typical usage scenario, I guess the user would access the `DisplayInfo` only via `DeviceDisplay.MainDisplayInfo`, which calls the constructor internally.

But you are right, in principle the end user can also construct a `DisplayInfo` variable manually. I will fix it like you proposed. Thanks for the review!

Reply:
Modified the code according to your suggestion and rebased ...